superset-frontend/packages/superset-ui-core/src/query/processFilters.ts

```diff
@@ -43,7 +43,6 @@ export default function processFilters(
   const { adhoc_filters, extras = {}, filters = [], where } = formData;
   const simpleWhere: QueryObjectFilterClause[] = filters;
 
-  const simpleHaving: QueryObjectFilterClause[] = [];
```

Comment:
discussion from [PR](https://github.com/apache/superset/pull/20338#issuecomment-1177284865), should remove druid simpleHaving clause